bio/gdc-client/download/wrapper.py

```
@@ -8,14 +8,15 @@
 from tempfile import TemporaryDirectory
 import glob
 
+extra = snakemake.params.get("extra", "")
 uuid = snakemake.params.get("uuid", "")
```

Ruff (0.15.13)

[error] Undefined name `snakemake`

(F821)